mne/io/meas_info.py

```diff
@@ -1211,3 +1214,141 @@ def _empty_info():
     info['nchan'] = info['sfreq'] = 0
     assert set(info.keys()) == set(RAW_INFO_FIELDS)
     return info
+
+
+class DigMontage(object):
+    """Montage for Digitized data
+
+    Montages are typically loaded from a file using read_montage. Only use this
+    class directly if you're constructing a new montage.
+
+    Parameters
+    ----------
+    hsp : array, shape (n_points, 3)
+        The positions of the channels in 3d.
```

Comment:
I think the idea will be to centralize how we add the digitizer info to the object. this will support Polhemus, BTi setup, and numpy.ndarrays. I think those are all the use cases. It has the ability to be expandable if there are other formats/systems to be added.